pa4/pa4_procedures.sql

```diff
@@ -0,0 +1,79 @@
+SET AUTOCOMMIT = ON;
+
+CREATE PROCEDURE select_by_substring_in_name_sproc(IN name VARCHAR(64))
+BEGIN 
+SET name = CONCAT('%', name, '%');
+SELECT b.id, b.name, b.year_date, b.dollar_price FROM books b 
+WHERE b.name LIKE name;
+END;
+drop procedure select_by_substring_in_name_sproc;
```

Comment:
How are you going to call the procedure if you delete it? I think it is not needed